packages/getMyPrescriptions/src/getMyPrescriptions.ts

```diff
@@ -67,6 +68,8 @@ const lambdaHandler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPro
         }
       }
     }
+    const nhsNumber = extractNHSNumber(event.headers["nhsd-nhslogin-user"])
+    logger.info(`nhsNumber: ${nhsNumber}`)
     const returnData = await spineClient.getPrescriptions(event.headers)
```

Comment:
We need to add the nhsNumber to this set of headers

Reply:
nhsNumber has been added to the headers